src/components/Calculator.js

```diff
@@ -0,0 +1,35 @@
+import React from 'react';
+
+// eslint-disable-next-line react/prefer-stateless-function
```

Comment:
- [ ] Rather than disable the Linters, strive to fix them as this makes you more accustomed to the style guide of the tech.

Reply:
@medaminedev66  In this part of code as a requirement of the project I have to use class-based components and when we use class components eslint shows this linter error that our components should be functional, so there is no way except to disable the linters.

Reply:
Hi @ahzamir, Could you please share your linter error with me?

Reply:
As I see, the linter that is disabled has a relation with the stateless component. Because your Calculator component is stateless, you should use `React.PureComponent` rather than `React.Component`.